packages/commons/src/fixtures/birth-bundle.ts

```diff
@@ -759,23 +759,7 @@ export const BIRTH_BUNDLE: SavedBundle<
             family: 'Mweene'
           }
         ],
-        extension: [
-          {
-            url: 'http://opencrvs.org/specs/extension/employee-signature',
-            valueSignature: {
-              type: [
-                {
-                  system: 'urn:iso-astm:E1762-95:2013',
-                  code: '1.2.840.10065.1.12.1.13',
-                  display: 'Review Signature'
-                }
-              ],
-              when: '2023-09-19T06:47:48.225Z',
-              contentType: 'image/png',
-              blob: 'data:image/png;base64,'
-            }
-          }
-        ],
```

Comment:
Instead of straight out removing this, let's put in how the data would look like now